vividus-plugin-ssh/build.gradle

```diff
@@ -8,7 +8,7 @@ dependencies {
     implementation(group: 'javax.inject', name: 'javax.inject', version: '1')
     implementation(group: 'org.slf4j', name: 'slf4j-api', version: versions.slf4j)
     implementation(group: 'org.apache.commons', name: 'commons-lang3', version: '3.12.0')
-    implementation(group: 'com.jcraft', name: 'jsch', version: '0.1.55')
+    implementation(group: 'com.github.mwiede', name: 'jsch', version: '0.2.6')
     implementation(group: 'com.jcraft', name: 'jsch.agentproxy.jsch', version: '0.0.9')
```

Comment:
https://github.com/mwiede/jsch#by-replacing-jsch-as-a-transitive-maven-dependency:
> *Addition*: You can further exclude any of `com.jcraft:jsch.agentproxy.jsch`, `com.jcraft:jsch.agentproxy.core` or `com.jcraft:jsch.agentproxy.pageant`, because these modules where integrated in this fork (see release notes of [0.1.66](https://github.com/mwiede/jsch/releases/tag/jsch-0.1.66)).